rules-tests/Php80/Rector/Class_/ClassPropertyAssignToConstructorPromotionRector/Fixture/property_with_sensitiveparameter_attribute.php.inc

```diff
@@ -0,0 +1,33 @@
+<?php
```

Comment:
please rename the file to `skip_property_promotion_with_sensitive_parameter_attribute.php.inc`